frontend/test/storybook/visual-regression/v-collection-header.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { test } from "@playwright/test"
+
+import breakpoints from "~~/test/playwright/utils/breakpoints"
+
+test.describe("VCollectionHeader", () => {
+  breakpoints.describeMobileAndDesktop(({ expectSnapshot }) => {
+    for (const languageDirection of ["ltr", "rtl"]) {
+      for (const collection of [
+        "tag",
+        "creator",
+        "source",
+        "source-with-long-name",
+      ]) {
```

Comment:
If we use the all collections story for this test instead then we don't need individual tests here. Is that worse for testing, though? I guess this way if only the tag changes then we have a tag-specific test that catches that change, rather than needing to dig through the diff to identify where the change is coming from.

Reply:
I refactored the story so that it's a single story for all 4 cases.